common/src/main/java/org/apache/falcon/metadata/MetadataMappingService.java

```diff
@@ -345,6 +345,9 @@ private void updateInstanceStatus(final WorkflowExecutionContext context) throws
         case IMPORT:
             updateImportedFeedInstance(context);
             break;
+        case EXPORT:
+            onFeedInstanceExported(context);
```

Comment:
Minor nit : Can we have uniform naming for similar methods?  This one should be updateExportedFeedInstance...